package.json

```diff
@@ -50,8 +50,7 @@
     "freedom": "^0.6.11"
   },
   "scripts": {
-    "test": "grunt test",
-    "prepublish": "grunt test",
+    "test": "grunt build jasmine",
```

Comment:
So, I changed the test line to prevent Travis running the integration tests (it runs `npm test`). This is not the only way, e.g. I could add a new target like `grunt test_headless` but felt that was overkill.